test/geo/wkb_test.exs

```diff
@@ -399,6 +399,13 @@ defmodule Geo.WKB.Test do
     assert(point.srid == 4326)
   end
 
+  test "Decode EWKB of empty MultiLineString" do
+    multi_line_string = Geo.WKB.decode!("010500000001000000010200000000000000")
+    assert multi_line_string.coordinates == [[]]
+    assert multi_line_string.properties == %{}
+    assert multi_line_string.srid == nil
```

Comment:
should we add a test to make sure the SRID is passed through too?

Reply:
Good idea. Just added it in dfe665fc0a45a2d7343a74e08046322464600b45